src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -2732,6 +2746,67 @@ public void setNgBrowserTest(NgBrowserTest ngBrowserTest) {
         this.ngBrowserTest = ngBrowserTest;
     }
 
+    public boolean isShadowDomHandlingEnabled(){
+        return shadowDomHandling;
+    }
+
+    public void setShadowDomHandlingTo(boolean shadowDomHandling){
+        this.shadowDomHandling = shadowDomHandling;
+    }
+
+    protected T getElementWithShadow(String place){
+        try{
+            if(place.startsWith("css=")) return (T) shadow.findElement(place.split("css=")[1]);
```

Comment:
It now requires the "ccs=" prefix to work with shadow. It is probably not the best way to do it and doesn't use the SeleniumHelper methods but I couldn't figure it out how to make that work. Open for better ideas.